src/modm/ui/color/rgbhtml.hpp

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2021, Thomas Sommer
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+#include "rgb.hpp"
+
+namespace modm::color::html
```

Comment:
You can do this since C++17 instead of nesting namespaces.

Reply:
Very nice.. also #pragma once